src/main/java/com/passbook/sparkeighteen/peristence/POJO/TransactionResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.passbook.sparkeighteen.peristence.POJO;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class TransactionResponse {
+
+    private Float amount;
```

Comment:
What if i want to see a specific  transaction ?
In response there is no transaction id?

Reply:
Added transaction id